modules/auxiliary/scanner/http/suite_crm_export_sqli.rb

```diff
@@ -0,0 +1,244 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::SQLi
+  include Msf::Exploit::Remote::HttpClient
+
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'SuiteCRM authenticated SQL injection in export functionality',
+        'Description' => %q{
+          This module exploits an authenticated SQL injection in SuiteCRM in versions before 7.12.6. The vulnerability
+          allows an attacker to send specically crafted requests to the export entry point of the application in order
+          to retrieve all the usernames and their associated password from the database.
+        },
+        'Author' => [
+          'Exodus Intelligence', # Advisory
+          'jheysel-r7', # poc + msf module
+          'Redouane NIBOUCHA <rniboucha@yahoo.fr>' # sql injection help
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['URL', 'https://blog.exodusintel.com/2022/06/09/salesagility-suitecrm-export-request-sql-injection-vulnerability/']
```

Comment:
Should add in a reference for https://docs.suitecrm.com/admin/releases/7.12.x/#_7_12_6